### PR TITLE
test(voltron): add unit tests for voltron renderer

### PR DESCRIPTION
**Pull request by BigLiao**
# Pre-PR Checklist

- [x] I added/updated relevant documentation.
- [x] I followed the [Convention Commit](https://conventionalcommits.org/) guideline with maximum 72 characters to submit commit message.
- [x] I squashed the repeated code commits.
- [x] I signed the [CLA].
- [x] I added/updated test cases to check the change I am making.
- [x] All existing and new tests are passing.

### CODE REVIEW

**Comment by tencent-adm** [comment]
[CLA assistant check](https://cla.opensource.tencent.com/Tencent/Hippy?pullRequest=2504) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.opensource.tencent.com/Tencent/Hippy?pullRequest=2504) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.opensource.tencent.com/check/Tencent/Hippy?pullRequest=2504) it.</sub>

**Review by lvfen — APPROVED** [review]

File: .github/workflows/voltron_render_tests.yml [hunk 1]
```diff
@@ -0,0 +1,24 @@
+name: '[voltron-render] unit tests'
```
Comment (zoomchan-cxj) [inline .github/workflows/voltron_render_tests.yml:1]: voltron-render 改成  voltron

File: renderer/voltron/analysis_options.yaml [hunk 2]
```diff
@@ -1,3 +1,26 @@
+#
+#
+# Tencent is pleased to support the open source community by making
+# Hippy available.
+#
+# Copyright (C) 2019 THL A29 Limited, a Tencent company.
```
Comment (zoomchan-cxj) [inline renderer/voltron/analysis_options.yaml:6]: license 的 2019 全部改成 2022

File: .github/workflows/voltron_render_tests.yml (hunk 1 above)
Comment (zoomchan-cxj) [inline .github/workflows/voltron_render_tests.yml:1]: [voltron-render ]改成  [voltron]

文件名 voltron_render_tests 改成 voltron_tests

**Review by skindhu — APPROVED** [review]

**Review by zoomchan-cxj — APPROVED** [review]
LGTM

**Pull request merged**